File: atv3/exerc3.py
class ContaBancaria:
    def __init__(self, titular, saldo_inicial):
        self.__saldo = saldo_inicial
        self.titular = titular

    def depositar(self, valor):
        self.__saldo += valor

    def sacar(self, valor):
        if valor <= self.__saldo:
            self.__saldo -= valor
        else:
            print("Saldo insuficiente.")

    def get_saldo(self):
        return self.__saldo

File: atv3/exerc3.go
package main
import "fmt"

type ContaBancaria struct {
    saldo   float64
    titular string
}

func (c *ContaBancaria) depositar(valor float64) {
    c.saldo += valor
}

func (c *ContaBancaria) sacar(valor float64) {
    if valor <= c.saldo {
        c.saldo -= valor
    } else {
        fmt.Println("Saldo insuficiente.")
    }
}

func (c ContaBancaria) getSaldo() float64 {
    return c.saldo
}
